ledger/catchpointtracker.go

```diff
@@ -1081,7 +1084,7 @@ func (ct *catchpointTracker) IsWritingCatchpointDataFile() bool {
 }
 
 // Generates a (first stage) catchpoint data file.
-func (ct *catchpointTracker) generateCatchpointData(ctx context.Context, accountsRound basics.Round, updatingBalancesDuration time.Duration) (uint64 /*totalAccounts*/, uint64 /*totalChunks*/, uint64 /*biggestChunkLen*/, error) {
+func (ct *catchpointTracker) generateCatchpointData(ctx context.Context, accountsRound basics.Round, updatingBalancesDuration time.Duration) (uint64 /*totalKVs*/, uint64 /*totalAccounts*/, uint64 /*totalChunks*/, uint64 /*biggestChunkLen*/, error) {
```

Comment:
I know you didn't write this, but I don't see why we shouldn't used named return values instead of these comments.

I dislike _naked returns_ that are sometimes used along with named return values. But the named return values themselves serve as useful documentation when the types are the same.

Reply:
@jannotti I have a similar view on naked returns.  Unless you prefer the thought below, I'll add the naked return with named return values.

I had considered returning a partially populated `catchpointFirstStageInfo` struct too because it's created in the subsequent processing step `recordFirstStageInfo`.  I held off because partially populated structs feel error prone to me, but I think it'd be considered a more Go-like approach.

Reply:
> I'll add the naked return with named return values.

I'm saying you can use named return values, and yet _not_ use naked returns.  This is fine:
```
func two(a, b int) (x int, y int) {
    return a+b, a-b
}
```

Reply:
@jannotti I attempted to bring in the feedback via https://github.com/algorand/go-algorand/pull/4808/commits/de8d3d78cfbba3a581e106926cbff7ffb9b93713#diff-9ca6833a989d244ce6b95aaddf4f2f0889203f7e3e6918b0c9ace37c62f21320R1087.  Can you have a look to see if it looks good to you?